README.txt

```diff
@@ -57,17 +57,21 @@ Installation
 Any platform
```

Comment:
I don't think there's any platform-specific code. I know it's been used on Windows as well as *nixes.